RF24Gateway.cpp

```diff
@@ -209,7 +209,12 @@ int ESBGateway<mesh_t, network_t, radio_t>::allocateTunDevice(char* dev, int fla
         // close(fd);
         //#if (RF24GATEWAY_DEBUG_LEVEL >= 1)
         std::cerr << "RF24Gw: Error: enabling TUNSETIFF" << std::endl;
-        std::cerr << "RF24Gw: If changing from TAP/TUN, run 'sudo ip link delete tun_nrf24' to remove the interface" << std::endl;
+        uint32_t UID = getuid();
+        if (UID) {
```

Comment:
I've been reading up on [`getuid()`](https://pubs.opengroup.org/onlinepubs/9699919799/functions/getuid.html). Am I to understand that a zero returned indicates super user or root?

Reply:
Yup, root is 0